system/Router/RouteCollection.php

```diff
@@ -728,7 +728,7 @@ public function getRedirectCode(string $from): int
 	 *     });
 	 *
 	 * @param string $name      The name to group/prefix the routes with.
-	 * @param mixed  ...$params
+	 * @param mixed|callable  ...$params
```

Comment:
Perhaps you can change it as below and see if no errors occur on PHPStorm?

```suggestion
	 * @param string            $name   The name to group/prefix the routes with.
	 * @param array|callable ...$params
```